examples_apps/run_tutorial_vibronic.py

```diff
@@ -5,7 +5,8 @@
 Vibronic spectra
 ================
 
-*Technical details are available in the API documentation:* :doc:`/code/api/strawberryfields.apps.vibronic`

```

Comment:
Instead of making these changes here @trbromley, can you instead make them to the other repo!